src/qibolab/instruments/qm/components/configs.py

```diff
@@ -42,3 +39,6 @@ class QmAcquisitionConfig(AcquisitionConfig):
     """
     offset: float = 0.0
     """Constant voltage to be applied on the input."""
+
+
+QmConfig = Union[OpxOutputConfig, QmAcquisitionConfig]
```

Comment:
Isn't this colliding with https://github.com/qiboteam/qibolab/blob/9c102a4feb29960af75fc2dda5f69b234563a0b8/src/qibolab/instruments/qm/config/config.py#L32

Do we really need it?

Reply:
I should change its name, but they are not truly colliding because of the different scope.

We don't strictly need it, because I could inline the `Union` somewhere else. But it's convenient, because I could register all the configurations types in one shot. I.e. if there will be any additional `Config` in the future, you should just add here and nowhere else.